### DISCUSSION TITLE
Loading combo failed

### DISCUSSION THREAD

**Discussion by Abdoxh9r** [DrPython3/MailRipV2#19] [Q&A] [answered]
i got this trying to load the combo !!!

**Comment by Freddielee21** [comment]
How can I get aws combos to crack smtp

**Reply by DrPython3** [reply]
No support for getting combos for using with Mail.Rip.

**Comment by jinxwoo13** [comment]
how can I fix this error?

![Screenshot_1](https://github.com/DrPython3/MailRipV2/assets/93089612/83a566a0-4826-432b-9756-43e2e93713bb)

**Comment by Sunnybee619** [comment]
I got an error message saying "combo fail" can you please tell me how to fix?

**Reply by DrPython3** [reply, marked as answer]
Check whether your combofile is encoded with UTF-8 ... you can use any tool like Notepad for that. And if so, split the combofile into smaller files to avoid any memory errors. Your combofile may also contain any characters that break the loading process. Check your file for that, too.
